extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -1157,6 +1138,15 @@ private void fireEvent(SecurityEvent.Type eventType, Map<String, Object> propert
         }
     }
 
+    private static String decryptIdToken(TenantConfigContext configContext, String idToken) {
+        if (configContext.oidcConfig().token().decryptIdToken().orElse(false)
```

Comment:
No point in using `Optional<Boolean>` if you're expecting it to be `false` if it's not set.